setup.py

```diff
@@ -1,53 +1,19 @@
 #!/usr/bin/env python
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-import ah_bootstrap
-import pkg_resources
-from setuptools import setup
-from astropy_helpers.setup_helpers import (register_commands, get_package_info,
-                                           add_exclude_packages)
-from astropy_helpers.version_helpers import generate_version_py
+# NOTE: most of the configuration is defined in setup.cfg
```

Comment:
```suggestion
# NOTE: most of the configuration, including the version number, 
# is defined in setup.cfg
```

because I think the most likely case here is someone looking here and wondering "how do I update the version?"